<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipParamDlg</class>
 <widget class="QDialog" name="RecipParamDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reciprocal Space Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="3">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Angles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>kfQ (deg):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit2Theta"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editKfQ"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>kiQ (deg):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editKiQ"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>2θ (deg):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>θ (deg):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editTheta"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Transfers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ΔE (meV):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editE"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Q (Å⁻¹):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editQ"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>q (Å⁻¹):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editq"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Initial Values</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>k_i (Å⁻¹):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editKi"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>E_i (meV):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editEi"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>λ_i (Å):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editLami"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>v_i (m/s):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editVi"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Final Values</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>k_f (Å⁻¹):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editKf"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>E_f (meV):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editEf"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>λ_f (Å):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editLamf"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>v_f (m/s):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editVf"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Scattering Vector</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QLineEdit" name="editQx"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="editQxrlu"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Q (Å⁻¹):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editQy"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Q (rlu):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="editQyrlu"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editQz"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="editQzrlu"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Reduced Scattering Vector</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="1">
       <widget class="QLineEdit" name="editqx"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="editqxrlu"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>q (Å⁻¹):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editqy"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>q (rlu):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="editqyrlu"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editqz"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="editqzrlu"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Lattice Vector</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="1">
       <widget class="QLineEdit" name="editGx"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="editGxrlu"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>G (Å⁻¹):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editGy"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>G (rlu):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="editGyrlu"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editGz"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="editGzrlu"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editKi</tabstop>
  <tabstop>editEi</tabstop>
  <tabstop>editLami</tabstop>
  <tabstop>editVi</tabstop>
  <tabstop>editKf</tabstop>
  <tabstop>editEf</tabstop>
  <tabstop>editLamf</tabstop>
  <tabstop>editVf</tabstop>
  <tabstop>editE</tabstop>
  <tabstop>editQ</tabstop>
  <tabstop>editq</tabstop>
  <tabstop>edit2Theta</tabstop>
  <tabstop>editTheta</tabstop>
  <tabstop>editKiQ</tabstop>
  <tabstop>editKfQ</tabstop>
  <tabstop>editQx</tabstop>
  <tabstop>editQy</tabstop>
  <tabstop>editQz</tabstop>
  <tabstop>editQxrlu</tabstop>
  <tabstop>editQyrlu</tabstop>
  <tabstop>editQzrlu</tabstop>
  <tabstop>editqx</tabstop>
  <tabstop>editqy</tabstop>
  <tabstop>editqz</tabstop>
  <tabstop>editqxrlu</tabstop>
  <tabstop>editqyrlu</tabstop>
  <tabstop>editqzrlu</tabstop>
  <tabstop>editGx</tabstop>
  <tabstop>editGy</tabstop>
  <tabstop>editGz</tabstop>
  <tabstop>editGxrlu</tabstop>
  <tabstop>editGyrlu</tabstop>
  <tabstop>editGzrlu</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RecipParamDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RecipParamDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
